src/components/LoginForm/index.js

```diff
@@ -38,12 +38,19 @@ const LoginForm = ({ onSubmit }) => {
         label={strings.SIGN_IN.email}
         keyboardType="email-address"
         autoCapitalize="none"
+        testID="email-input"
```

Comment:
Will this also work on Android or should we make a wrapper?

Reply:
The wrapper is only needed if we want to do e2e. By the time we reach to that point the bug with android will probably be fixed.